Add Northwestern Polytechnical University(NWPU, China) with their CS faculty

reasonW opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

reasonW committed:
add  Northwestern Polytechnical University(NWPU)

reasonW committed:
add faculties of Northwestern Polytechnical University(NWPU),CHINA

add 14 faculties of Northwestern Polytechnical University(NWPU),CHINA

reasonW committed:
Updated the faculty's DBLP name

Updated the faculty's DBLP name and removed the currently unclear faculty

reasonW committed:
Merge pull request #1 from reasonW/NWPU-patch-1

add  Northwestern Polytechnical University(NWPU)

emeryberger commented:
I see a lot of Google Scholar links instead of home pages. Please replace these with something else, e.g., their official pages from the institution.

reasonW committed:
Replace Google pages of some faculties with personal homepages

reasonW commented:
Thanks for your reminder, i have tried my best to replace some faculties' Google Scholar links.

reasonW committed:
add some faculties of Northwestern Polytechnical University 

Those faculties with 'NOSCHOLARPAGE' didn‘t create and maintain their google scholar entries.

emeryberger committed:
removed spurious space and newline
